json/liste.json

```diff
@@ -35,6 +35,11 @@
         "Mozilla Italia Hub Bot": "https://github.com/Sav22999/mozitahub_bot",
         "Vademecum Generale, Tecnico e Common Voice": "https://github.com/MozillaItalia/firefox-vademecum"
     },
+    "link_vademecum": {
+        "Vademecum Common Voice": "https://github.com/MozillaItalia/firefox-vademecum/raw/master/volantino/pdf/Vademecum_2.0_CV.pdf",
+        "Vademecum Generale": "https://github.com/MozillaItalia/firefox-vademecum/raw/master/volantino/pdf/Vademecum_2.0_VG.pdf",
+        "Vademecum Tecnico": "https://github.com/MozillaItalia/firefox-vademecum/raw/master/volantino/pdf/Vademecum_2.0_VT.pdf"
```

Comment:
Un'unica cosa: c'è un altro modo in cui riusciamo a prendere sempre il solito link a prescindere dal filename?